### PR TITLE
Bump flutter_map from 0.9.0 to 0.10.1+1

### PR DESCRIPTION
**Pull request by Xennis**
Refers to #46

### CODE REVIEW

**Comment by fryette** [comment]
@lpongetti could you please publish a new version?

**Comment by sm2017** [comment]
@lpongetti could you please publish a new version?

File: example/pubspec.yaml [hunk 1]
```diff
@@ -20,7 +20,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  flutter_map: ^0.9.0
+  flutter_map: ^0.10.1
```
Comment (fryette) [inline example/pubspec.yaml:23]:
```suggestion
  flutter_map: ^0.10.1+1
```

File: pubspec.yaml [hunk 2]
```diff
@@ -13,7 +13,7 @@ dependencies:
   flutter:
     sdk: flutter
 
-  flutter_map: ^0.9.0
+  flutter_map: ^0.10.1
```
Comment (fryette) [inline pubspec.yaml:16]:
```suggestion
  flutter_map: ^0.10.1+1
```

Comment (Xennis) [reply pubspec.yaml]: Added a second commit on top for that.

File: example/pubspec.yaml (hunk 1 above)
Comment (Xennis) [reply example/pubspec.yaml]: Added a second commit on top for that.

**Comment by pro100svitlo** [comment]
is it really so hard to review this pr? 😥

**Comment by pro100svitlo** [comment]
25 days to check 2 lines...
unbelievable.
you have made a great addition to the `flutter_map`. THANK YOU for that, honestly!
But if you don't have enough time to support it, maybe you can transfer this repo to the `flutter_map` owner?

**Comment by pro100svitlo** [comment]
@lpongetti please 🙏

**Comment by fryette** [comment]
@pro100svitlo you can refer on this commit instead of waiting author

**Comment by pro100svitlo** [comment]
> @pro100svitlo you can refer on this commit instead of waiting for author

Thank you 🙏 
I  know about it. 
My main point is asking to transfer this repo to the `flutter_map`, cos this situation with updates is pretty constant 😞

**Comment by Xennis** [comment]
Dear all,

maybe I can turn this discussion here in a different way: 

The original issues #46 is already quite old. In that issue and in the MR here there wasn't a single life vital sign of the maintainer. I think it's not like he is not willing to react he simply (for unknown reasons) currently can't. Let's hope the best. Linking him again and again does not help.

People start to use my fork also does not really help. I'm not the maintainer of it and I don't give you any guarantee I won't break your build (i.e. by force pushing, deleting the branch, ...).

Similar to @pro100svitlo I only see the options to transfer that repo. In general, I think it would boost flutter_map if the plugin maintainers give access to more people to their code and ensure a higher quality. Of course on a voluntary basis. https://github.com/fluttercommunity is a great example for that. Each package has a clear owner but multiple people can work on all the packages.

The question is if the owner of https://github.com/fleaflet is happy having more repos there or if not (and a new "community" organization) is needed).

At least I'm maintaining https://github.com/Xennis/flutter_map_location and would be happy to transfer that to a group where I still have the ownership for it but more people are allowed to merge request.

Regards
Xennis

**Comment by Xennis** [comment]
I brought up the idea above in https://github.com/fleaflet/flutter_map/issues/932

**Pull request closed without merging**